api-report/container-runtime.api.md

```diff
@@ -147,7 +147,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     // Warning: (ae-forgotten-export) The symbol "GCNodeType" needs to be exported by the entry point index.d.ts
     getNodeType(nodePath: string): GCNodeType;
     // (undocumented)
-    getPendingLocalState(): unknown;
+    getPendingLocalState(): PendingLocalState;
```

Comment:
we should avoid modifying the public surface area. i assume this is to resolve the casting problem. I didn't realize it was being use in a different package, but we can still use the type without exporting it. 

to do that you will need to add an index.ts file into the test folder of this packages, and export PendingLocalState from there, along with everything else exported in the src/index.ts. then in the test file you will import "@fluidframework/container-runtime/test". this will produce a lint error you should suppress due to submodule importing. submodule importing shouldn't geneally be used, but in the case where we only need something for test in another one of our packages its fine, as we control both sides. we use the same technique in the merge tree package, so you can find an example there.

the above will only work for the type you are adding PendingLocalState, for the method we should still return unknown, and the test can cast to PendingLocalState directly, which is safer than an `any` cast, as if the type changes, we will still get a compile error.